### PR TITLE
Implements chunk support

### PR DESCRIPTION
**Pull request by cascornelissen**

### CODE REVIEW

File: svg-spritemap.js [hunk 1]
```diff
@@ -14,28 +16,74 @@ function SVGSpritemapPlugin(options) {
         glob: {},
         prefix: 'sprite-',
         gutter: 2,
-        filename: 'spritemap.svg'
+        filename: 'spritemap.svg',
```
Comment (gijsvanzon) [inline svg-spritemap.js]: Better use output, instead of filename. This isn't clear where filename is used for.

File: svg-spritemap.js [hunk 2]
```diff
@@ -94,25 +142,24 @@ SVGSpritemapPlugin.prototype.apply = function(compiler) {
...
-                callback();
-            });
+    compiler.plugin('emit', function(compilation, callback) {
+        compilation.chunks.forEach(function(chunk) {
+            if ( chunk.name !== options.chunk ) {
+                return;
+            }
+
+            // Remove entry (.js file) from compilation assets since it's empty
+            delete compilation.assets[chunk.files[0]]
```
Comment (gijsvanzon) [inline svg-spritemap.js]: missing semicolon

File: svg-spritemap.js (hunk 1 above)
Comment (cascornelissen) [inline svg-spritemap.js]: That would make this BIC and this follows the setup used in plugins like [`extract-text-webpack-plugin`](https://github.com/webpack-contrib/extract-text-webpack-plugin#options).

File: README.md [hunk 3]
```diff
@@ -23,42 +23,24 @@ module.exports = {
...
-| `gutter`   | `2`               | Amount of pixels added between each sprite to prevent overlap                                     |
-| `filename` | `'spritemap.svg'` | Name for the generated file (located at the webpack `output.path`)                                |
+| Option          | Default           | Description                                                                                                    |
+| --------------- | ----------------- | -------------------------------------------------------------------------------------------------------------- |
+| `src`           | `'**/*.svg'`      | [`glob`](http://npmjs.com/package/glob) used for finding the SVGs that should be in the spritemap              |
+| `glob`          | `{}`              | Options for [`glob`](http://npmjs.com/package/glob#options)                                                    |
+| `svgo`          | `{}`              | Options for [`SVG Optimizer`](http://npmjs.com/package/svgo), pass `false` to disable                          |
+| `svg4everybody` | `false`           | Options for [`SVG4Everybody`](https://www.npmjs.com/package/svg4everybody#usage),  pass `false` to disable     |
```
Comment (gijsvanzon) [inline README.md]: Pass true to enable. Default option is false

**Pull request merged**